rest/importtest/import_test.go

```diff
@@ -2301,6 +2301,12 @@ func TestImportRollbackMultiplePartitions(t *testing.T) {
 	bucket := base.GetTestBucket(t)
 	defer bucket.Close(ctx)
 
+	numVBuckets, err := bucket.GetMaxVbno()
+	require.NoError(t, err)
+	if numVBuckets != 1024 {
+		t.Skipf("Test requires 1024 vBuckets, but only had %d", numVBuckets)
```

Comment:
```suggestion
		t.Skipf("Test requires 1024 vBuckets, but only had %d, will fix in CBG-4544", numVBuckets)
```

just so it is easier to find the test from the code, and to know we have a ticket where we are planning to fix when reading the code.